api/package.json

```diff
@@ -65,9 +65,9 @@
 	],
 	"scripts": {
 		"build": "tsc --build && copyfiles \"src/**/*.{yaml,liquid}\" -u 1 dist",
-		"cli": "NODE_ENV=development SERVE_APP=false ts-node --script-mode --transpile-only src/cli/run.ts",
-		"dev": "NODE_ENV=development SERVE_APP=false ts-node-dev --files --transpile-only --respawn --watch \".env\" --inspect=0 --exit-child -- src/start.ts",
-		"start": "npx directus start",
+		"cli": "NODE_ENV=development SERVE_APP=false node --require @swc-node/register src/cli/run.ts",
+		"dev": "NODE_ENV=development SERVE_APP=false ts-node-dev --transpile-only --respawn --watch \".env\" --inspect=0 -- src/start.ts",
```

Comment:
I wonder if the `dev` script could make use of SWC 🤔 Applying it solely to `cli` seems like a very limited use

Reply:
Agreed! Lets either use this for all dev stuff, or not at all 🙂

Reply:
> I wonder if the dev script could make use of SWC 🤔

It is 😃👍 Unfortunately not directly via the SWC register because that caused some issues, but via `ts-node`: https://github.com/directus/directus/pull/16609/files#diff-8381d7ec5e0a6228d02ec0c0f6f801bcb1b38c8f7fcaff16e9351a46e91321a2R16-R19